src/routes/safe/components/Transactions/TxsTable/columns.tsx

```diff
@@ -30,15 +29,15 @@ const NOT_AVAILABLE = 'n/a'
 interface AmountData {
   decimals?: number | string
   symbol?: string
-  value: number | string
+  value: string
 }
 
 const getAmountWithSymbol = (
   { decimals = 0, symbol = NOT_AVAILABLE, value }: AmountData,
   formatted = false,
 ): string => {
-  const nonFormattedValue = new BigNumber(value).times(`1e-${decimals}`).toFixed()
-  const finalValue = formatted ? formatAmount(nonFormattedValue).toString() : nonFormattedValue
+  const humanReadableAmount = humanReadableTokenAmount(value, decimals as number)
```

Comment:
can we convert `decimals` to number, instead of using `as number`? Something like `+decimals`?